webrtc.html

```diff
@@ -11090,19 +11090,73 @@ <h3 id="mediastreamtrack-network-use">MediaStreamTrack</h3>
-        <p>The basics of <code>MediaTrackSupportedConstraints</code>,
-        <code>MediaTrackCapabilites</code>, <code>MediaTrackConstraints</code>
-        and <code>MediaTrackSettings</code> is outlined in [[!GETUSERMEDIA]].
-        However, the <code>MediaTrackSettings</code> for a
-        <code>MediaStreamTrack</code> sourced by an
-        <code><a>RTCPeerConnection</a></code> will only be populated with
-        members to the extent that data is supplied by means of the remote
-        <code><a>RTCSessionDescription</a></code> applied via
-        <code>setRemoteDescription</code> and the actual RTP data. This means
-        that certain members, such as <code>facingMode</code>,
-        <code>echoCancellation</code>, <code>latency</code>,
-        <code>deviceId</code> and <code>groupId</code>, will always be
-        missing.</p>
+        <p>The concept of constraints and constrainable properties, including
+        <code>MediaTrackConstraints</code>
+        (<code>MediaStreamTrack.getConstraints()</code>,
+        <code>MediaStreamTrack.applyConstraints()</code>), and
+        <code>MediaTrackSettings</code>
+        (<code>MediaStreamTrack.getSettings()</code>) are outlined in
+        [[!GETUSERMEDIA]]. However, the constrainable properties of tracks
+        sourced from an <code>RTCPeerConnection</code> are different than those
+        sourced by <code>getUserMedia()</code>. Only constraints explicitly
+        specified in the context of WebRTC are applicable.
+        The settings of a track sourced from an <code>RTCPeerConnection</code>
+        represent the currently received frames/samples after downsampling based
+        on applied constraints, if applied.
+        <code>MediaStreamTrack.getCapabilities()</code> MUST always return the
+        empty set.</p>
+        <p>The following constrainable properties are defined to apply only to
+        remote video <code><a>MediaStreamTrack</a></code> objects generated by
+        an <code>RTCPeerConnection</code>:</p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Property Name</th>
+              <th>Values</th>
+              <th>Notes</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="def-constraint-width">
+              <td><dfn>width</dfn></td>
+              <td><code><a>ConstrainULong</a></code></td>
+              <td>The width or width range, in pixels. The constrainable
+              property allows downscaling. As a setting, this is the width of
+              the latest frame.</td>
+            </tr>
+            <tr id="def-constraint-height">
+              <td><dfn>height</dfn></td>
+              <td><code><a>ConstrainULong</a></code></td>
+              <td>The height or height range, in pixels. The constrainable
+              property allows downscaling. As a setting, this is the height of
+              the latest frame.</td>
```

Comment:
Clarification needed: If latest frame had height 800, but we are downscaling to 600, should getSettings() say 600 or 800?